adoc/admin-ses-integration.adoc

```diff
@@ -14,623 +14,541 @@ You can check this by running `rpm -q ceph-common` and `rpm -q xfsprogs`.
 
-. Create a pod that uses the persistent volume claim.
+. Create the Persistent Volume Claim:
 +
+====
+  kind: PersistentVolumeClaim
+  apiVersion: v1
+  metadata:
+    name: ceph-rbd-sc
+  spec:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 2Gi // <1>
+====
+<1> The request volume size. Reference to link:https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#setting-requests-and-limits-for-local-ephemeral-storage[Setting requests and limits for local ephemeral storage] to see supported suffixes.
++
+[NOTE]
+====
+Deleting Persistent Volume Claim does not remove RBD volume in the Ceph cluster.
+====
 
-----
-kubectl apply -f - << *EOF*
-apiVersion: v1
-kind: Pod
-metadata:
-  name: <POD_NAME>
-spec:
-  containers:
-  - name: <CONTAINER_NAME>
-    image: <IMAGE_NAME>
-    volumeMounts:
-    - name: rbdvol
-      mountPath: /mnt/rbdvol
-      readOnly: false
-  volumes:
-  - name: rbdvol
-    persistentVolumeClaim:
-      claimName: <PVC_NAME>
-*EOF*
-----
-. Verify that the pod exists and check its status.
+. Create the Pod:
 +
+====
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: ceph-rbd-sc
+  spec:
+    containers:
+    - name:  ceph-rbd-sc
+      image: busybox
+      command: ["sleep", "infinity"]
+      volumeMounts:
+      - mountPath: /mnt/ceph_rbd // <1>
+        name: volume
+    volumes:
+    - name: volume
+      persistentVolumeClaim:
+        claimName: ceph-rbd-sc // <2>
+====
+<1> The volume mount path inside the Pod.
+<2> The Persistent Volume Claim name.
 
-----
-kubectl get pod
-----
-. Once the pod is running, check the volume:
+. Once the pod is running, check the volume is mounted:
 +
 
 ----
-kubectl exec -it <CONTAINER_NAME> -- df -k ...
-/dev/rbd3               999320      1284    929224   0% /mnt/rbdvol
-...
+kubectl exec -it pod/ceph-rbd-sc -- df -k | grep rbd
+  Filesystem     1K-blocks    Used Available Use% Mounted on
+  /dev/rbd0        1998672    6144   1976144   1% /mnt/ceph_rbd
 ----
 
 
-In case you need to delete the pod, run the following command:
+=== Using CephFS in Pods
 
-----
-kubectl delete pod <CONTAINER_NAME>
-----
 
-.Deleting A Pod
-[NOTE]
-====
-When you delete the pod, the persistent volume claim is deleted as well.
-The RBD is not deleted.
-====
-
-=== Using CephFS in a Pod
+The procedure below describes how to use CephFS in Pod.
 
 
-The procedure below describes steps to take when you need to use a CephFS in a pod.
+.Procedure: Using CephFS In Pods
 
-.Procedure: Using CephFS In A Pod
 
-
-. Retrieve the Ceph admin secret.
-You can get the key value using the following command:
+. link:https://docs.ceph.com/en/latest/rbd/rados-rbd-cmds/#create-a-block-device-user[Create a Block Device User] to use as CephFS admin and record the key:
 +
 ----
-ceph auth get-key client.admin
+ceph auth get-or-create-key client.myCephFSAdmin mds 'allow *' mgr 'allow *' mon 'allow *' osd 'allow *'  | tr -d '\n' | base64
```

Comment:
that seems overly broad. Please limit this to the least privileges necessary

Reply:
Since the user should have a better idea of how the filesystem should be restricted, so I've included a reference link in `[NOTE]`. What's your recommended change?

Reply:
I would like to see an example where a dedicated fs is created with a user, that is only allowed to operate on this filesystem.

Reply:
Thanks for the hint, addressed in https://github.com/SUSE/doc-caasp/pull/1010/commits/75e1df874a08117df40fd2447153341ad30a5a2c.

Reply:
better, thank you